### ISSUE TITLE
Erlang apk installation error

### ISSUE THREAD

**Issue by sandytrinh** [msaraiva/alpine-erlang#3]
When installing Erlang package in alpine linux via this command line

```
apk add --update erlang --update-cache --allow-untrusted -X http://dl-4.alpinelinux.org/alpine/edge/community/
```

I got the following error:

```
ERROR: unsatisfiable constraints:
so:libncurses.so.6 (missing):
required by: erlang-18.1-r3[so:libncurses.so.6]
```

**Comment by sandytrinh** [comment]
@msaraiva here's the results for the commands you have requested:

`cat /etc/alpine-release` returns `3.2.3`
`cat /etc/apk/repositories` returns `http://dl-4.alpinelinux.org/alpine/v3.2/main`

**Comment by ncopa** [comment]
you need either use both edge/main and edge/community repo or wait for v3.3.

Erlang should work with this as your repos:

```
 http://dl-4.alpinelinux.org/alpine/edge/main
 http://dl-4.alpinelinux.org/alpine/edge/community
```

The v3.3 main and community repos should have erlang in a week or so.

**Comment by msaraiva** [comment]
@sandytrinh have you tried to change your `/etc/apk/repositories` as proposed? Does it work now?

As soon we have Alpine Linux v3.3, I'll update all images as described [here](https://github.com/msaraiva/alpine-erlang/issues/2). After that, we don't expect this kind of issues anymore.

@ncopa, thanks for your response. It's great to hear that version 3.3 is right at the door.

**Comment by sandytrinh** [comment]
@msaraiva Sorry for the late response. I can confirm that @ncopa's suggestion works! Thanks @ncopa!

Thanks for taking a look at this guys :)

I look forward to the new release.

**Comment by ncopa** [comment]
erlang should now be available from v3.3/community repo.

**Comment by sandytrinh** [comment]
@ncopa Awesome!
